server/models/Keys.js

```diff
@@ -8,6 +8,8 @@ class Keys {
   usageCount;
   createdAt;
   updatedAt;
+  apiKey;
+  createDate;
```

Comment:
These are added as the frontend used these to show the date in the table. I first modified frontend code, then felt to keep it as it is and add keys in the schema. Same for the next 2 comments as well.

Reply:
That's now what I am asking. The key which you added are useless. There were already existing. So why did you add new keys ?

Reply:
Because the keys are used in frontend before i wrote the code. They are present in the backend, but the keys used in frontend earlier were different. So either i could have changed the keys used in frontend or add keys in backend.

Reply:
Can you join a session and explain.